scale_build/packages/package.py

```diff
@@ -173,11 +173,15 @@ def checkout(self, branch_override=None):
             run(['git', '-C', self.source_path, 'fetch', '--unshallow'], logger=git_logger, check=False)
             run(['git', '-C', self.source_path, 'fetch', 'origin', branch], logger=git_logger)
             run(['git', '-C', self.source_path, 'reset', '--hard', f'origin/{branch}'], logger=git_logger)
+            run(['git', '-C', self.source_path, 'submodule', 'update', '--checkout', '--depth=1'])
```

Comment:
I imagine you forgot `logger=git_logger` here so I am going to ask again: what is the joy of having to pass logger each and every time?